utils/extension.ts

```diff
@@ -12,6 +13,20 @@ export const getAddress = async (address: string) => {
   }
 }
 
+export const getSelectedAccount = (accounts: WalletAccount[]) => {
```

Comment:
Where is this used?

Reply:
I wanted to use this in WalletModal.vue when we are setting wallet `l:135`, should I remove it?